packages/components/src/components/text-field-wrapper/text-field-wrapper-styles.ts

```diff
@@ -63,6 +69,15 @@ export const getComponentCss = (
           ...(isNumber && {
             MozAppearance: 'textfield', // hides up/down spin button for Firefox
           }),
+          ...((isCalendar || isTime) && {
+            // for native placeholder color in safari, background ahs to be a special value, `transparent` or `rgba(0,0,0,0)` won't work
+            // this works nice for `theme="light"`, but for `theme="dark"` placeholder appears blue which is still better
+            // than having invisible dots or colons for data/time or not seeing the value at all after selection
+            '@supports (-webkit-hyphens: none)': {
+              background: 'rgba(0,0,1,0)',
+              color: isThemeDark(theme) ? getThemedColors(theme).primaryColor : 'initial',
```

Comment:
Does the WebKit VRT test show the new behaviour?

Reply:
No, the safari screenshots don't show any placeholder at all and didn't do so before (which would have been with wrong colors.

<img width="428" alt="Bildschirmfoto 2024-03-05 um 08 28 57" src="https://github.com/porsche-design-system/porsche-design-system/assets/4380295/e919c407-0861-41eb-b416-aa16eb334b54">